src/lib/cache.js

```diff
@@ -48,19 +48,19 @@ class CacheClient {
     }
   }
 
-  get (key) {
+  async get (key) {
     if (enabled) {
-      return catboxMemoryClient.get(key)
+      return await catboxMemoryClient.get(key)
```

Comment:
Can't we return the promise directly here? We got recommendation from nodeJS expert regarding this and he suggested not to use await in this case. Return promise directly.

Reply:
Ok. I thought this might of been the reason why tests were failing first run and I just left it as is after fixing the real problem.

Reverted here and throughout.